src/plugins/theme/fontFamily.js

```diff
@@ -10,5 +10,6 @@ module.exports = function () {
     stanford: ['Stanford','"Source Serif Pro"', 'Georgia', 'Times','"Times New Roman"', 'serif'],
     fab: ['"Font Awesome 5 Brands"'],
     far: ['"Font Awesome 5 Free"'],
+    fas: ['"Font Awesome 5 Solid"'],
```

Comment:
See above comment

Reply:
Huh, yeah, but with different class names... Now why did they do that...

https://github.com/FortAwesome/Font-Awesome/blob/master/scss/solid.scss
https://github.com/FortAwesome/Font-Awesome/blob/master/scss/regular.scss

Reply:
I know....crazy right? 😂

Reply:
Ah, cuz the names of the icons are different. hmm. kay.